tutorials/pages/shimmerevm-setup.md

```diff
@@ -0,0 +1,68 @@
+# Setup ShimmerEVM and run Solidity Smart Contract
```

Comment:
This sounds misleading to me. `Shimmer EVM` will be a public SC chain deployed and validated by the IF and partners. A better fit for the title could be `Connect to Shimmer EVM Testnet and deploy a Solidity Smart Contract`.